packages/cli/src/commands/validator/status.test.ts

```diff
@@ -8,27 +10,93 @@ process.env.NO_SYNCCHECK = 'true'
 
 const KNOWN_DEVCHAIN_VALIDATOR = '0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f'
 
-// Status needs to be re written as it calls L1 only methods on contracts
-
-testWithAnvilL2('validator:status', (web3: Web3) => {
+testWithAnvilL1('validator:status', (web3: Web3) => {
   const writeMock = jest.spyOn(ux.write, 'stdout')
   const logMock = jest.spyOn(console, 'log')
 
+  // In anvil the blocks don't have any extraData on them
+  // This is a bare minimum needed by status command to work
+  jest.spyOn(IstanbulUtils, 'parseBlockExtraData').mockImplementation(() => {
+    return {
+      parentAggregatedSeal: {
```

Comment:
do alfajores blocks have this extra stuff? 

it looks in the code like we are assuming yes

Reply:
it's a L1 test to make sure the command is backwards compatible, but in anvil we just don't have `extraData` field, so it had to be mocked

for L2 we're not using it and `extraData` is removed from the blocks

it was used by this code: https://github.com/celo-org/developer-tooling/pull/447/files#diff-a7df5a1df25f113949adb3b3398fdf0e7117da6923a43b7a67e2204feb99eb99R135-R137

Reply:
```bash

$ cast block --rpc-url https://alfajores-forno.celo-testnet.org

(...)
extraData            0x
(...)